docs/data/date-pickers/date-time-picker/date-time-picker.md

```diff
@@ -12,63 +12,90 @@ materialDesign: https://m2.material.io/components/date-pickers
 
-- The `MobileNextDateTimePicker` component works best for touch devices and small screens.
-- The `DesktopNextDateTimePicker` component works best for mouse devices and large screens.
+- The `DesktopNextDateTimePicker` component which works best for mouse devices and large screens.
+  It renders the views inside a popover and allows editing values directly inside the field.
 
-By default, the `NextDateTimePicker` component renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
-This can be customized with the `desktopModeMediaQuery` prop.
+- The `MobileNextDateTimePicker` component which works best for touch devices and small screens.
+  It renders the view inside a modal and does not allow editing values directly inside the field.
 
-There are certain caveats when testing pickers, please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for more information.
+- The `NextDateTimePicker` component which will render `DesktopNextDateTimePicker` or `MobileNextDateTimePicker` depending on the device it runs on.
+
+  By default, it renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
+  This can be customized with the `desktopModeMediaQuery` prop.
 
 {{"demo": "ResponsiveDateTimePickers.js"}}
 
+:::warning
+There are certain caveats when testing pickers, please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for more information.
+:::
+
 ## Static mode
 
-It is also possible to render any date time picker without the modal/popover and text field.
-This will enable building custom popover/modal containers.
+It is also possible to render the Date Time Picker without the popover/modal and field.
+This can be helpful when building custom popover/modal containers.
 
 {{"demo": "StaticDateTimePickerDemo.js", "bg": true}}
 
 ## Form props
 
-The date time picker component can be disabled or read-only.
+The Date Time Picker component can be disabled or read-only.
 
 {{"demo": "FormPropsDateTimePickers.js"}}
 
+## Views
+
+The Date Time Picker component can contain six views: `day`, `month`, `year`, `hours`, `minutes` and `seconds`.
+By default, only the `year`, `day`, `hours`, and `minutes` views are enabled.
+
+You can customize the enabled views using the `views` prop.
+Views will appear in the order they're included in the `views` array.
+
+{{"demo": "DateTimePickerViews.js"}}
+
+## Landscape orientation
+
+By default, the Date Time Picker component automatically sets the orientation based on the `window.orientation` value.
+
+You can force a specific orientation using the `orientation` prop.
+
+{{"demo": "StaticDateTimePickerLandscape.js", "bg": true}}
+
 ## Validation
 
 You can find the documentation in the [Validation page](/x/react-date-pickers/validation/)
 
-## Customization
```

Comment:
This demo was a mixed bag of props with nothing in common, most of them being now properly documented elsewhere